.github/workflows/main.yml

```diff
@@ -64,18 +63,20 @@ jobs:
             smalltalk: Pharo32-alpha
           - os: macos-latest
             smalltalk: Pharo32-3.0
+          - os: macos-latest
+            smalltalk: GemStone64-3.5.8
+          - os: macos-latest
+            smalltalk: GemStone64-3.6.8
           - os: windows-2019
             smalltalk: Pharo64-10
           - os: windows-2019
             smalltalk: Pharo64-6.0
-          - os: windows-2019
-            smalltalk: GemStone64-3.5.3
           - os: windows-2019
             smalltalk: GemStone64-3.5.8
           - os: windows-2019
-            smalltalk: GemStone64-3.6.0
+            smalltalk: GemStone64-3.6.8
           - os: windows-2019
-            smalltalk: GemStone64-3.6.5
+            smalltalk: GemStone64-3.7.1
     continue-on-error: ${{ matrix.os == 'macos-latest' && startsWith(matrix.smalltalk, 'GemStone64') }}
```

Comment:
You can remove the `continue-on-error` property if the GS builds are fixed.

Reply:
so I did fix the continue-on-error and magically in my latest run (with the continue-on-error removed) the Squeak32 job is passing :)